src/utils/formatDollarAmt.tsx

```diff
@@ -26,7 +26,7 @@ export const formatAmount = (num: number | undefined, digits = 2) => {
   if (num === 0) return '0'
   if (!num) return '-'
   if (num < 0.001) {
-    return '$<0.001'
+    return '<0.001'
```

Comment:
This fn is actually unused. Could you update your PR to just remove it (L24 and on)?